test/testUtils.ts

```diff
@@ -0,0 +1,28 @@
+import Color from '../src/util/color';
```

Comment:
Please move to `test/unit/lib/expect-extensions.ts` or some other file you think has a better name in that folder.

Reply:
`import type Color...`?

Reply:
Moved to `test/unit/testUtils.ts`.
Definitely not  `import type Color...`. A js reference to `Color` class is needed.

Reply:
File name convention is with "_" I think...

Reply:
My bad, corrected.